### PR TITLE
update about page

### PR DESCRIPTION
**Pull request by samundra**
#### Changes
- [x] add github link
- [x] update doc to follow 80 char limit

### CODE REVIEW

**Comment by hardfire** [comment]
LGTM

File: about.md [hunk 1]
```diff
@@ -3,22 +3,40 @@ layout: page
...
-1. Presentation time for hands on workshop or demo type presentation/talk will be dealt in a case by case basis.
+1. Decision on the presentation sequence and time done by organizing group will
+be final.
+1. Presentation time for hands on workshop or demo type presentation/talk will
+be dealt in a case by case basis.
+
+## Want to update this document
+If you feel the above information is insufficient then you can always send us
+pull request. Feel free to fork and update this document. Our github repository
+is [https://github.com/developers-nepal/php](https://github.com/developers-nepal/php).
```
Comment (geshan) [inline about.md:42]: Can we add a link to the FB group @samundra ?

Comment (geshan) [inline about.md:42]: and yes the slack channel too :)

Comment (samundra) [inline about.md:42]: added in https://github.com/developers-nepal/php/pull/42 PR

**Pull request merged**